requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
Because it is the way we distribute software for Fedora/CentOS/RHEL. Most people (myself included) would not like to install things via pip or other language based package managers.

Do you not see value in building an RPM ?